prog/main.go

```diff
@@ -152,28 +154,50 @@ func (c *containerLabelFiltersFlag) String() string {
 }
 
 func (c *containerLabelFiltersFlag) Set(flagValue string) error {
-	newAPITopologyOption, err := c.toAPITopologyOption(flagValue)
+	c.filterIDBase = fmt.Sprintf(c.filterIDBase+"%d", c.filterNumber)
```

Comment:
This doesn't do what you want. I think you want a local variable instead of overwriting the base.

Optional: keep using the length instead of a counter like you did before.